### ISSUE TITLE
gpodder-sync of the listened episodes is synced from device 1 to Podfetch but not to device 2

### ISSUE THREAD

**Issue by update-freak** [SamTV12345/PodFetch#1060]
### Describe the bug

I currently have the problem that the gpodder-sync of the listened episodes is synced from device 1 to Podfetch but not to device 2.

I created a docker container with the following docker compose 

```
services:
podfetch:
image: samuel19982/podfetch:latest
container_name: PodFetch
user: 1026:100
ports:
- 4135:8000
volumes:
- /volume1/docker/podfetch/podcasts:/app/podcasts:rw
- /volume1/docker/podfetch/db:/app/db:rw
environment:
- POLLING_INTERVAL=300
- SERVER_URL=https://podfetch.beispiel.dynv6.net/
- DATABASE_URL=sqlite:///app/db/podcast.db
- BASIC_AUTH=true
- USERNAME=Felix
- PASSWORD=L#xmUgqw$Wi9&F@H
- GPODDER_INTEGRATION_ENABLED=true
restart: unless-stopped
```

After that I did the following steps:

1. Adding a user: docker exec -it DOCKER-ID /app/podfetch users add
2. Restart the container
3. Login with the Felix-User
4. Add all podcasts from AntennaPod
5. Logout with Felix and login with felix
6. PodFetch -> Profil -> API-Key ► generate ► save
7. use AntennaPod on device 1 to login and force sync to the new created device Podcasts
8. open AntennaPod on device 2 and sync also with device Podcasts

Expectation is that when I listen to a episode on device 1 it's also listened on device 2 and vice versa.

### Reproduction

see description

### System Info

```Shell
Synology NAS DSM 7.2 with Docker
```

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/SamTV12345/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/SamTV12345/.github/blob/main/CONTRIBUTING.md).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by SamTV12345** [comment]
I don't know what the user flow is. For me it was always. I have two separate devices. I want to listen to different podcasts on different devices. I could change this behavior. That way every device is synced with the other one. Currently it is just as you described Device 1 -> Webview, Device2 -> Webview and not Device1->Device2.

**Comment by update-freak** [comment]
yes, that explains it :D
My user flow is indeed a bit different: 2 devices: 1 main smartphone and a cheaper 2nd smartphone for biking or the beach.
Therefore I want to know which of the podcasts on my 1st smartphone I already listened when I go biking or relaxing on the beach (and did not want to use my better 1st smartphone)

**Comment by SamTV12345** [comment]
Yes :) . Try to add both devices as one in antennapod. So in the selection just choose the first device. It should hopefully work so both devices are in sync and grab the updates from the other one.

**Comment by update-freak** [comment]
All right. This I already tried: created the device name "Podcasts" when synchronise with the first device and then I also used the existing device "Podcasts" for the second device -> this did not work for me. Could you please doublecheck if this also did not work for you (Maybe I have to check it also again)?

**Comment by update-freak** [comment]
I checked this issue again (with fresh docker container). It's still not possible to sync the listened episodes on podfetch to the 2nd smartphone
